lib/protoflow-blocks/src/blocks/sys/write_socket.rs

```diff
@@ -0,0 +1,185 @@
+extern crate std;
+use super::SysBlocks;
+use crate::prelude::{vec, String};
+use crate::{IoBlocks, StdioConfig, StdioError, StdioSystem, System};
+use core::str::FromStr;
+use protoflow_core::{
+    Block, BlockError, BlockResult, BlockRuntime, InputPort, Message, SystemBuilding,
+};
+use protoflow_derive::Block;
+use serde::{Deserialize, Serialize};
+use simple_mermaid::mermaid;
+use std::{
+    format,
+    net::TcpStream,
+    sync::{Arc, Mutex, PoisonError},
+};
+use tracing::error;
+/// A block that writes a proto object to a TCP port.
```

Comment:
Leave a newline prior to doc comments (and run `cargo fmt`!)

Reply:
done